### PR TITLE
Close to enemies

### PR DESCRIPTION
**Pull request by Kelghor**
Hello, 

There is scenario COOP "Close to enemies"  for 1-6 players on 7 rounds - Version 1.4 - discused on dicord forum.

When i developed, i compilated only my scenario, because, compilate Draftbook didnt works for me.  So i hope my push will be ok and not screw up compilations. 

There was needed to add whole coop chapter.

### CODE REVIEW

**Comment by qwrtln** [comment]
If you're doing it on Windows, enable symlinks in the git repo: https://stackoverflow.com/questions/5917249/git-symbolic-links-in-windows/59761201#59761201
Then, in the `draft-scenarios` dir you can compile like this: `latexmk -pdflua -shell-escape drafts.tex`
Or, use the `tools/drafts.sh` script.

File: draft-scenarios/metadata.tex [hunk 1]
```diff
@@ -472,6 +472,8 @@
 
 \include{campaigns/main.tex}
 
+\include{coop/main.tex}
```
Comment (qwrtln) [inline draft-scenarios/metadata.tex:475]: Please maintain the ordering of the original. Coops  go before clash. Also, name the dir `coops`, like in the main book.

File: draft-scenarios/coop/main.tex [hunk 2]
```diff
@@ -0,0 +1,7 @@
+\addscenariogroup{Coop}{\layout/coops.png}
+
+\clearpage
+
+\input{draft-scenarios/coop/close_to_enemies.tex}
+
+
```
Comment (qwrtln) [inline draft-scenarios/coop/main.tex:7]:
```suggestion
\input{coops/close_to_enemies.tex}
```
Please pay attention to how it's done in other directories.

**Comment by qwrtln** [comment]
Fix the issues shown in the comments. Once you do, the book should compile both here and on your machine.

File: po4a.cfg [hunk 3]
```diff
@@ -3,6 +3,7 @@
 
 [type: latex] sections/title_page.tex $lang:sections/translated/$lang/title_page.tex
 
+[type: latex] coops/close_to_enemies.tex $lang:coops/translated/$lang/close_to_enemies.tex
```
Comment (qwrtln) [inline po4a.cfg:6]:
```suggestion
```
No need for that. We don't generate translations for draft scenarios.

**Comment by Kelghor** [comment]
Closed and will be reopened as PR of project member

**Pull request closed without merging**